rpc/src/v1/impls/traces.rs

```diff
@@ -144,4 +145,40 @@ impl<C, M> Traces for TracesClient<C, M> where C: BlockChainClient + 'static, M:
 				Ok(Value::Null)
 			})
 	}
+
+	fn send_raw_transaction(&self, params: Params) -> Result<Value, Error> {
+		try!(self.active());
+		trace!(target: "jsonrpc", "call: {:?}", params);
+		from_params::<(Bytes, _)>(params)
+			.and_then(|(raw_transaction, flags)| {
+				let raw_transaction = raw_transaction.to_vec();
+				let flags: Vec<String> = flags;
+				let analytics = CallAnalytics {
+					transaction_tracing: flags.contains(&("trace".to_owned())),
+					vm_tracing: flags.contains(&("vmTrace".to_owned())),
+					state_diffing: flags.contains(&("stateDiff".to_owned())),
+				};
+				match UntrustedRlp::new(&raw_transaction).as_val() {
+					Ok(signed) => {
+						let r = take_weak!(self.client).call(&signed, analytics);
+						if let Ok(executed) = r {
+							// TODO maybe add other stuff to this?
+							let mut ret = map!["output".to_owned() => to_value(&Bytes(executed.output)).unwrap()];
```

Comment:
I agree with Tomek here. It should be a serializable struct. Imo it's a scope of this PR.